etl/steps/data/garden/hmd/2024-11-19/hfd.meta.yml

```diff
@@ -0,0 +1,288 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  others:
+    bo_1: |-
+      <% if birth_order == '1' %>
+       first
```

Comment:
Why do you use the space here? It seemed for natural for me to add the space below (`of {definitions.others.bo_1}`)

Reply:
because there is the case when `birth_order=='total'`. In that case I want it to be empty.

If I used the line `of {definitions.others.bo_1}) ..` I'd have two empty space AFAIK.